examples/create-react-app/src/components/App.js

```diff
@@ -0,0 +1,100 @@
+import { Badge, Button, Dialog, TabPanel, Tabs } from '@cmsgov/design-system';
+import React, { useState } from 'react';
+import usflag from '../images/us_flag_small.png';
+
+function App() {
+  const [showModal, setShowModal] = useState(false);
+
+  return (
+    <div className="ds-base">
+      <header className="ds-u-padding--3 ds-u-sm-padding--6 ds-u-display--block ds-u-fill--primary-darkest">
+        <h1 className="ds-u-margin--0 ds-u-color--white ds-u-font-size--display ds-u-text-align--center">
+          Hello, world!
+        </h1>
+        <center>
+          <Badge variation="info" size="big">
+            <img className="c-usa-banner__header-flag" src={usflag} alt="U.S. flag" />
+            &nbsp;CMS Design system
+          </Badge>
+        </center>
+      </header>
+
+      <Tabs>
+        <TabPanel id="summary" tab="Summary">
```

Comment:
This is really great Ni! I like how you put in content here. I wonder if we can simplify this though, so the content doesnt detract from the example. Mayb we can just remove the other Tabs besides summary?

Reply:
I thought the Tabs component would be fun to use. What do you think if I just keep it to 2 tabs - summary and uswds tabs only?

Reply:
Ok that’s fine, i just think we should be intentional with how we create examples. Generally i’m not a huge fan of arbitrary or verbose example content, as it doesn’t really provide the user with anything. in this example, usage of react component is the most important thing, and that doesn’t have to be achieved with tabs

Reply:
the child design system example is a good example of this. originally we had random style changes, but as soon as we tried to the example realistic, we ran into many problems, and we found the need to provide a better example of organization and overrides